### PR TITLE
home work

### PR DESCRIPTION
**Pull request by wahhab1404**
I tried but cant solve it :(

### CODE REVIEW

**Comment by JackieCasper** [comment]
Hey, looks like there isn't much done here. This will be marked as incomplete for now. Take another look and add on before tomorrow at midnight for credit!

File: index.js [hunk 1]
```diff
@@ -1,31 +1,51 @@
...
+
     // changes background color of green and yellow to black
-}
+    $('#slowLight').css('background-color','black');
+    $('#goLight').css('background-color','black');
+  }
+}    
 
 // turn on slow function
-function turnOnSlowLight() {
```
Comment (wahhab1404) [inline index.js:8]: by mistake :(

**Comment by JackieCasper** [comment]
Looks good 👍

File: index.js [hunk 2]
```diff
@@ -1,24 +1,51 @@
 // turn on stop function 
 function turnOnStopLight() {
+  $(this).css('background-color', 'red')
```
Comment (JackieCasper) [inline index.js:3]: lines 2 and 3 aren't needed

**Pull request closed without merging**